content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -3234,6 +3234,34 @@ Set a timestamp of the time when a record is created.
 
 - Implemented at Prisma level
 
+#### Examples
+
+##### Set current timestamp value when a record is created
+
+<TabbedContent tabs={[<FileWithIcon text="Relational databases" icon="database"/>, <FileWithIcon text="MongoDB" icon="database"/>]}>
+<tab>
+
+```prisma
+model User {
+  id        String   @id @default(uuid())
+  createdAt DateTime @default(now())
+}
+```
+
+</tab>
+<tab>
+
+```prisma
+model User {
+  id        String   @id @default(uuid()) @map("_id")
+  createdAt DateTime @default(now())
+}
+```
+
+</tab>
+</TabbedContent>
```

Comment:
I think we can do away with the `TabbedContent` here since the behavior & syntax is the same in both relational dbs & MongoDB

Reply:
I just look at all the existing content.

I don't think the example would be valid for MongoDB.

Reply:
I don't have a super strong opinion here but would generally tend to include an ID (so that the entire model is valid). For relational DBs I'm ok with just using `@id` but for MongoDB we should be using the full blurb that's needed because (afaiu) that's really the only way an ID for MongoDB can be defined.

Reply:
by the way, I opened another issue to double-check whether `cuid()` and `uuid()` are even valid as default values on MongoDB: https://github.com/prisma/docs/issues/4766